universal-login-relayer/src/core/services/execution/messages/PendingMessages.ts

```diff
@@ -27,14 +27,14 @@ export default class PendingMessages {
     await this.addSignatureToPendingMessage(messageHash, message);
     const status = await this.getStatus(messageHash);
     status.messageHash = messageHash;
```

Comment:
Unnecessary line

Reply:
Agree, looks unnecessary.
But if it is in fact necessary, then I feel that `getStatus` should be modified to include `messageHash` in the status

Reply:
Also note that this was not introduced in this PR.

Reply:
But still, we can remove it